matching_values:
  x_therm: 30
  y_therm: 30
  c_x: 6.785
  c_y: 7
